pkgs/misc/vim-plugins/overrides.nix

```diff
@@ -249,6 +252,18 @@ self: super: {
   ncm2-ultisnips = super.ncm2-ultisnips.overrideAttrs(old: {
     dependencies = with super; [ ultisnips ];
   });
+  
+  notational-fzf-vim = super.notational-fzf-vim.overrideAttrs(old: {
+    dependencies = with self; [ fzf-vim ];
+    patchPhase = ''
+      substituteInPlace plugin/notational_fzf.vim \
+        --replace "'rg'" "'${ripgrep}/bin/rg'" \
+        --replace \
+        "let s:python_executable = executable('pypy3') ? 'pypy3' : 'python3'" \
+        "let s:python_executable = '${python3}/bin/python3'"
+    '';
+    buildInputs = [ ripgrep python3 ];
```

Comment:
if these are being patched in, I don't you need them to be in buildInputs. Due to contextual strings, the related packages will be added to the derivation/runtime closure.
```suggestion
```

Reply:
Okay.